src/TransformableImage.js

```diff
@@ -0,0 +1,205 @@
+import React, { PureComponent, PropTypes } from 'react';
+import { View, Text, Image, ViewPropTypes } from 'react-native';
+import ViewTransformer from 'react-native-view-transformer';
+
+export default class TransformableImage extends PureComponent {
+    static propTypes = {
+        source: PropTypes.oneOfType([
+            PropTypes.shape({ uri: PropTypes.string.isRequired }),
+            PropTypes.number
+        ]).isRequired,
+        style: ViewPropTypes ? ViewPropTypes.style : View.propTypes.style,
+        onLoad: PropTypes.func,
+        onLoadStart: PropTypes.func,
+        dimensions: PropTypes.shape({
+            width: PropTypes.number,
+            height: PropTypes.number
+        }),
+        enableTransform: PropTypes.bool,
+        enableScale: PropTypes.bool,
+        enableTranslate: PropTypes.bool,
+        onTransformGestureReleased: PropTypes.func,
+        onViewTransformed: PropTypes.func,
+        imageComponent: PropTypes.func,
+        resizeMode: PropTypes.string,
+        errorComponent: PropTypes.func
```

Comment:
errorComponent is only for getSize error, which is fine but can be mistaken with Image error.
Not sure how to resolve it yet.

Reply:
Well, I don't really know what kind of other error we could catch, except for the `getSize` one which will happen if the URL is broken.

If this is actually the only error that could happen, I think the prop's name is fine.

Reply:
I was thinking that user can interpret it as a error component that will be shown on Image loading error.